Hc.Gosia/Hc.Gosia.Js.Ex3/questionaire.js

```diff
@@ -0,0 +1,42 @@
+﻿var questionaire = questionaire || {};
+
+(function () {
+    var shuffle = function (tab) {
+        for (var i = tab.length; i; i--) {
+            var j = Math.floor(Math.random() * i);
+            [tab[i - 1], tab[j]] = [tab[j], tab[i - 1]];
+        }
+    }
+
+    var finishRequest = function (json) {
+        return json.results.map(item=> {
+            var question = {
+                text: item.question,
+                answears: [{ text: item.correct_answer, isCorrect: true }]
```

Comment:
zrobiłem błąd w tym słowie :) powinno być 'answer' :) Może sobie to poprawisz lokalnie w każdym miejscu ? Dobre ćwiczenie na refaktoryzację